maven-plugins/helidon-archetype-maven-plugin/src/main/java/io/helidon/build/maven/archetype/IntegrationTestMojo.java

```diff
@@ -157,6 +157,15 @@
     private Map<String, String> externalValues;
 
     /**
+<<<<<<< HEAD
```

Comment:
Looks like a bad merge op. Not sure why the pipeline didn't detect that.

Reply:
fixed